packages/fxa-payments-server/src/routes/Subscriptions/Cancel/CancelSubscriptionPanel.tsx

```diff
@@ -202,30 +203,37 @@ const CancelSubscriptionPanel = ({
         {!cancelRevealed ? (
           <>
             <div className="with-settings-button">
-              <div className="price-details" data-testid="price-details">
-                <PriceDetails
-                  total={intervalPriceDetailsData.invoiceDisplayTotal}
-                  tax={intervalPriceDetailsData.taxAmount}
-                  showTax={intervalPriceDetailsData.showInvoiceTax}
-                  currency={plan.currency}
-                  interval={plan.interval}
-                  intervalCount={plan.interval_count}
-                  className="price-details plan-pricing"
-                  dataTestId="price-details-standalone"
-                />
-                <Localized
-                  id={nextBillL10nId}
-                  vars={nextBillL10nVars}
-                  elems={{
-                    strong: <span className="font-semibold"></span>,
-                  }}
+              <div className="flex items-start">
+                {promotion_name && (
+                  <img src={couponIcon} alt="" className="ltr:pr-2 rtl:pl-2" />
+                )}
+                <span
+                  className="flex flex-col flex-4 text-grey-900 ltr:pr-8 rtl:pl-8"
+                  data-testid="price-details"
                 >
-                  <div data-testid="sub-next-bill">
-                    Your next bill of{' '}
-                    <span className="font-semibold">{nextBillAmount}</span> is
-                    due <span className="font-semibold">{nextBillDate}</span>
+                  <div className="font-semibold mb-2">
+                    {promotion_name && (
+                      <span data-testid="sub-coupon-applied">
+                        {promotion_name} coupon applied:&nbsp;
```

Comment:
This string needs to be localized.

Reply:
For posterity, it was discussed that while localization is currently not necessary for Monitor, other products are available outside of the US and utilize coupons. Therefore, we decided to localize the string.